uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -387,6 +394,48 @@ public void setMqttTimeoutSec(int mqttTimeoutSec) {
         log.info("MQTT timeout set to {} second(s)", mqttTimeoutSec);
     }
 
+    /**
+     * Sets MQTT content type to transmit.
+     *
+     * @param contentType MQTT content type to transmit
+     */
+    @And("I set MQTT 'content type' to {string} to transmit")
+    public void setMqttTxContentType(String contentType) {
+        this.txContentType = contentType;
+        log.info("MQTT content type set to {} to transmit", contentType);
+    }
+
+    /**
+     * Sets MQTT content type to receive.
+     *
+     * @param contentType MQTT content type to receive
+     */
+    @And("I set MQTT 'content type' to {string} to receive")
```

Comment:
Please use something similar to 
    @And("I set the 'retain' flag in expected received messages to {booleanOrNullValue}")

Reply:
The step was renamed in 41edf65c896419d5e3fc6a8684a8ffd6126319b9